test/clients/test_clients.py

```diff
@@ -30,5 +30,9 @@ def test_it_finds_clients(id_base):
     assert clients.find(client_id1) == client1
     assert clients.find(client_id2) == client2
 
+    for client in clients:
+        assert isinstance(client, Client)
+        assert client in clients
```

Comment:
😦

Reply:
`Clients` has an overloaded `__contains__` which is exercised by this check

Reply:
Ah, in that case I’d check it in `test_it_contains_clients`.